### ISSUE TITLE
Drawings in bonsai does not print fonts properly, unless the font is manually installed on a system level. 

### ISSUE THREAD

**Issue by Slackking1** [IfcOpenShell/IfcOpenShell#5433]
Either this is something that can be fixed in the bonsai plugin, or maybe just a note in the getting started documentation. 
Considered just adding a pull on the documentation, but I'm still getting into this project and it's structure.

Running blender 4.2.1, bonsai 0.8.0, inkscape 1.3.2, svg opened through edge browser

If you don't install the font from the bonsai plugin folder you get the following.
![without fonts installed](https://github.com/user-attachments/assets/d9b9524d-4f00-4648-aa7c-1f6e9b72c3ab)

When installed through windows you get this on drawing print. 
![with fonts installed](https://github.com/user-attachments/assets/02feb3ec-c30b-4b9b-a000-418403055b02)

So in short it seems exporting to .svg is dependent on system fonts.
